apps/src/code-studio/pd/workshop_dashboard/reports/foorm/results.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Tab, Tabs} from 'react-bootstrap';
+import SectionResults from './section_results';
+
+export default class Results extends React.Component {
+  static propTypes = {
+    questions: PropTypes.object.isRequired,
+    thisWorkshop: PropTypes.object.isRequired,
+    sessions: PropTypes.arrayOf(PropTypes.string).isRequired,
+    courseName: PropTypes.string
+    // workshopRollups: PropTypes.object,
```

Comment:
nit: if committing commented-out code it's good to have a comment explaining why :)